src/platform/site-wide/user-nav/selectors.js

```diff
@@ -13,6 +13,6 @@ export const selectUserGreeting = createSelector(
       return startCase(toLower(name.first || sessionFirstName));
     }
 
-    return email;
+    return 'My Account';
```

Comment:
If we're never going to use `email`, then we can probably remove the email selector.